nixos/modules/system/boot/loader/grub/grub.nix

```diff
@@ -224,7 +224,11 @@ in
 
       extraConfig = mkOption {
         default = "";
-        example = "serial; terminal_output.serial";
```

Comment:
Was the previous `terminal_output.serial` actually wrong?

It seems so to me: My grub2 console does not accept `terminal_output.serial`, only `terminal_output.serial` (with space instead of dot).

This would be useful to know, because there's still a use of that on current `master`, here: https://github.com/NixOS/nixpkgs/blob/44a502bf6d9c929b4d63ef65bdff4785a7543222/nixos/tests/installer.nix#L32

Reply:
Code originally introduced in https://github.com/NixOS/nixpkgs/commit/a65b5ec81c699b21fc635ce909332dcda4346027#r38681077.

Reply:
As far as I know `terminal_output.serial` isn't going to work.  That's what led me to writing up this change in the first place - the example didn't work when I tried it, and the grub docs indicated the syntax I replaced it with.

Reply:
@benley Would you be up for PRing a fix for the test i linked as well then?

Reply:
Sure thing: https://github.com/NixOS/nixpkgs/pull/85832